### PR TITLE
GH-219 Updated Version to 1.0.1 in code 

### PR DESCRIPTION
**Pull request by milindmore22**
and Updated settings options. for latest GA ID.
related to #219

### CODE REVIEW

File: inc/classes/class-config.php [hunk 1]
```diff
@@ -60,6 +63,20 @@ protected function setup_options() {
 				]
 			);
 		}
+
+		// Update version.
+		if ( false !== $current_version && REVENUE_GENERATOR_VERSION !== $current_version && REVENUE_GENERATOR_VERSION > $current_version ) {
+
+			// Update new version.
+			update_option( 'lp_rg_version', REVENUE_GENERATOR_VERSION );
```
Comment (mangesh) [inline inc/classes/class-config.php:71]: Bump the version after actually updating the settings. Probably after the line number `77`.

**Comment by milindmore22** [comment]
@mangesh Addressed PR feedback please review again.

**Comment by milindmore22** [comment]
@mangesh Updated code as per your suggestion, please re-review.

File: inc/classes/class-settings.php [hunk 2]
```diff
@@ -33,10 +33,12 @@ protected function __construct() {
...
 	protected function setup_options() {
 
-		// Fresh install.
-		if ( false === get_option( 'lp_rg_settings_options' ) ) {
-			// @todo, make region and currency empty and let the merchant choose, once EU is ready on upstream.
-			// Set default global options.
+		$settings_options = get_option( 'lp_rg_settings_options' );
+
+		// Fresh install or We don't have all the options.
+		if ( false === get_option( 'lp_rg_settings_options' ) || ( ! empty( $settings_options ) && is_array( $settings_options ) && count( $settings_options ) < 4 ) ) {
```
Comment (mangesh) [inline inc/classes/class-settings.php:39]: Non-blocker: You can make this more readable by putting each condition on a new line.

**Comment by milindmore22** [comment]
@mangesh addressed PR feedback please review again.

**Pull request merged**